src/TcoCore/src/TcoCore.Wpf/Converters/SignalToBrushConverter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Media;
+using Vortex.Presentation.Wpf.Converters;
+
+namespace TcoCore
+{
+    public class SignalToBrushConverter : BaseConverter
+    {
+        public override object ToConvert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            try
+            {
+                var signal = (bool)value;
+                if (signal)
+                {
+                    return Application.Current.TryFindResource("SignalOnBrush") ?? Brushes.GreenYellow;
```

Comment:
I'd like to discuss usage of a resource here... It's good idea to have same color for digital input in ie. a Cylinder, and here... an app that uses wpf, should define preferably in app.xaml a resource with this name.

Reply:
I'd go with resources from the material-design library's colors (Primary/Secondary/Error) and maybe we can add some other resources ('Alert/OnAlert (to capture attention), Active/OnActive (to indicate activity). 
Not a great eye for aesthetics... so open to other ideas :)

Reply:
Ok so I managed to find possible brushes in material toolkit that we could use...we'll see along the way.